rust/src/ctr.rs

```diff
@@ -1,12 +1,12 @@
 //! `ctr.rs`: Counter Mode encryption/decryption (128-bit IV size)
 //!
-//! TODO: this whole module is a legacy wrapper around aesni's former internal
+//! TODO: this whole module is a legacy wrapper around aes's former internal
```

Comment:
FWIW, this is actually describing a legacy API found only in the `aesni` crate. But the whole comment could probably be rewritten to say it should be switched to the `stream-cipher` crate when it becomes available.

Reply:
You should be able to edit my PR, do you want to change it to something you like more?

Reply:
I'll circle back on it when I do some other documentation changes